### DISCUSSION TITLE
Avast Free AV flags FC v1.46 as malware

### DISCUSSION THREAD

**Discussion by Ghastleigh** [Rem0o/FanControl.Releases#1547] [General]
I just updated Avast (program and database) and after restart it went nuts because Fancontrol.sys apparently wants too mess with drivers on a kernel level or something like that.

Also, virus total found this:
![image](https://user-images.githubusercontent.com/8240257/219372614-6b3ca7c2-cb55-4cca-b5be-719b10f92bc7.png)

Both look like some false positives tho.

**Comment by Rem0o** [comment]
https://github.com/Rem0o/FanControl.Releases/issues/1521#issuecomment-1431918992
